qa-tests-backend/scripts/run-compatibility.sh

```diff
@@ -22,10 +23,11 @@ set -euo pipefail
 
 compatibility_test() {
     require_environment "SENSOR_CHART_VERSION"
+    require_environment "CENTRAL_CHART_VERSION"
```

Comment:
[nit] Please make central version go before sensor version in all places touched in this PR. It's just more idiomatic in this repo to mention central before sensor when both things are mentioned near to each other.

Reply:
Done